src/app/services/actions/query-action-creators.ts

```diff
@@ -128,17 +129,16 @@ export function toggleRSCPopup(query: IQuery): Function {
       return authenticatedRequest(dispatch, query).then(async (response: Response) => {
         const result = await parseResponse(response, respHeaders);
         for (const i of result.value) {
-          // check if Sample Teams App is installed
           if (i.teamsApp.externalId === TEAMS_APP_ID) {
```

Comment:
I know this wasnt part of your task, but is it possible to use a better variable name than i? Just for it to be easier to understand what each item in the list might be

Reply:
c7da6b2f138edd68423247c86ae7ac624f3f6875